<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Maxime Lucas </title> <meta name="author" content="Maxime Lucas"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="science, research, physics, biology, networks"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://maximelucas.github.io/publications/"> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Maxime</span> Lucas </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">research </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">code </a> </li> <li class="nav-item "> <a class="nav-link" href="/vitae/">vitae </a> </li> <li class="nav-item "> <a class="nav-link" href="/contact/">contact </a> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <script src="/assets/js/bibsearch.js?3e2e6ee9cb3229a69412dc9dab540159" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <p>The symbol ‘*’ indicates that the authors contributed equally.</p> <h2 id="preprints">preprints</h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2024topological" class="col-sm-8"> <div class="title">Topological analysis of brain dynamical signals indicates signatures of seizure susceptibility</div> <div class="author"> <em>M. Lucas</em>, D. Francois, C. Donato, A. Skupin, and D. Proverbio </div> <div class="periodical"> 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.48550/arXiv.2412.01911" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Epilepsy is known to drastically alter brain dynamics during seizures (ictal periods), but its effects on background (non-ictal) brain dynamics remain poorly understood. To investigate this, we analyzed an in-house dataset of brain activity recordings from epileptic zebrafish, focusing on two controlled genetic conditions across two fishlines. After using machine learning to segment and label recordings, we applied time-delay embedding and Persistent Homology – a noise-robust method from Topological Data Analysis (TDA) – to uncover topological patterns in brain activity. We find that ictal and non-ictal periods can be distinguished based on the topology of their dynamics, independent of genetic condition or fishline, which validates our approach. Remarkably, within a single wild-type fishline, we identified topological differences in non-ictal periods between seizure-prone and seizure-free individuals. These findings suggest that epilepsy leaves detectable topological signatures in brain dynamics even outside of ictal periods. Overall, this study demonstrates the utility of TDA as a quantitative framework to screen for topological markers of epileptic susceptibility, with potential applications across species.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@unpublished</span><span class="p">{</span><span class="nl">lucas2024topological</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Topological analysis of brain dynamical signals indicates signatures of seizure susceptibility}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Francois, D. and Donato, C. and Skupin, A. and Proverbio, D.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv:2412.01911}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.48550/arXiv.2412.01911}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{brain}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2024functional" class="col-sm-8"> <div class="title">Functional reducibility of higher-order networks</div> <div class="author"> <em>M. Lucas</em>, L. Gallo, A. Ghavasieh, F. Battiston, and M. De Domenico </div> <div class="periodical"> 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.48550/arXiv.2404.08547" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://github.com/maximelucas/hypergraph_reducibility" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="abstract hidden"> <p>Empirical complex systems are widely assumed to be characterized not only by pairwise interactions, but also by higher-order (group) interactions that affect collective phenomena, from metabolic reactions to epidemics. Nevertheless, higher-order networks’ superior descriptive power – compared to classical pairwise networks – comes with a much increased model complexity and computational cost. Consequently, it is of paramount importance to establish a quantitative method to determine when such a modeling framework is advantageous with respect to pairwise models, and to which extent it provides a parsimonious description of empirical systems. Here, we propose a principled method, based on information compression, to analyze the reducibility of higher-order networks to lower-order interactions, by identifying redundancies in diffusion processes while preserving the relevant functional information. The analysis of a broad spectrum of empirical systems shows that, although some networks contain non-compressible group interactions, others can be effectively approximated by lower-order interactions – some technological and biological systems even just by pairwise interactions. More generally, our findings mark a significant step towards minimizing the dimensionality of models for complex systems</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@unpublished</span><span class="p">{</span><span class="nl">lucas2024functional</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Functional reducibility of higher-order networks}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Gallo, L. and Ghavasieh, A. and Battiston, F. and De Domenico, M.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv:2404.08547}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.48550/arXiv.2404.08547}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <h2 id="journal-articles">journal articles</h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="moriame2025hamiltonian" class="col-sm-8"> <div class="title">Hamiltonian control to desynchronize Kuramoto oscillators with higher-order interactions</div> <div class="author"> Martin Moriamé, <em>Maxime Lucas</em>, and Timoteo Carletti </div> <div class="periodical"> <em>Physical Review E</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1103/PhysRevE.111.044307" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Synchronization is a ubiquitous phenomenon in nature. Although it is necessary for the functioning of many systems, too much synchronization can also be detrimental, e.g., (partially) synchronized brain patterns support high-level cognitive processes and bodily control, but hypersynchronization can lead to epileptic seizures and tremors, as in neurodegenerative conditions such as Parkinson’s disease. Consequently, a critical research question is how to develop effective pinning control methods capable to reduce or modulate synchronization as needed. Although such methods exist to control pairwise-coupled oscillators, there are none for higher-order interactions, despite the increasing evidence of their relevant role in brain dynamics. In this work, we fill this gap by proposing a generalized control method designed to desynchronize Kuramoto oscillators connected through higher-order interactions. Our method embeds a higher-order Kuramoto model into a suitable Hamiltonian flow, and builds up on previous work in Hamiltonian control theory to analytically construct a feedback control mechanism. We numerically show that the proposed method effectively prevents synchronization. Although our findings indicate that pairwise contributions in the feedback loop are often sufficient, the higher-order generalization becomes crucial when pairwise coupling is weak. Finally, we explore the minimum number of controlled nodes required to fully desynchronize oscillators coupled via an all-to-all hypergraphs.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">moriame2025hamiltonian</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Hamiltonian control to desynchronize Kuramoto oscillators with higher-order interactions}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Moriam\'e, Martin and Lucas, Maxime and Carletti, Timoteo}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physical Review E}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{111}</span><span class="p">,</span>
  <span class="na">issue</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{044307}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{American Physical Society}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.111.044307}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="robiglio2025synergistic" class="col-sm-8"> <div class="title">Synergistic signatures of group mechanisms in higher-order systems</div> <div class="author"> T. Robiglio, M. Neri, D. Coppes, C. Agostinelli, F. Battiston, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'M. Lucas, G. Petri' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Physical Review Letters</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1103/PhysRevLett.134.137401" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>The interplay between causal mechanisms and emerging collective behaviors is a central aspect of understanding, controlling, and predicting complex networked systems. In our work, we investigate the relationship between higher-order mechanisms and higher-order behavioral observables in two representative models with group interactions: a simplicial Ising model and a social contagion model. In both systems, we find that group (higher-order) interactions show emergent synergistic (higher-order) behavior. The emergent synergy appears only at the group level and depends in a complex, nonlinear way on the trade-off between the strengths of the low- and higher-order mechanisms and is invisible to low-order behavioral observables. Our work sets the basis for systematically investigating the relation between causal mechanisms and behavioral patterns in complex networked systems with group interactions, offering a robust methodological framework to tackle this challenging task.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">robiglio2025synergistic</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Synergistic signatures of group mechanisms in higher-order systems}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Robiglio, T. and Neri, M. and Coppes, D. and Agostinelli, C. and Battiston, F. and Lucas, M. and Petri, G.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physical Review Letters}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{134}</span><span class="p">,</span>
  <span class="na">issue</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{137401}</span><span class="p">,</span>
  <span class="na">numpages</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevLett.134.137401}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="nurisso2025higherorder" class="col-sm-8"> <div class="title">Higher-order Laplacian Renormalization</div> <div class="author"> M. Nurisso, M. Morandini, <em>M. Lucas</em>, F. Vaccarino, T. Gili, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'G. Petri' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Nature Physics</em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41567-025-02784-1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://github.com/nplresearch/higher_order_LRG" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="abstract hidden"> <p>The renormalization group is a pillar of the theory of scaling, scale invariance and universality in physics. Recently, this tool has been adapted to complex networks with pairwise interactions through a scheme based on diffusion dynamics. However, as the importance of polyadic interactions in complex systems becomes more evident, there is a pressing need to extend the renormalization group methods to higher-order networks. Here we fill this gap and propose a Laplacian renormalization group scheme for arbitrary higher-order networks. At the heart of our approach is the introduction of cross-order Laplacians, which generalize existing higher-order Laplacians by allowing the description of diffusion processes that can happen on hyperedges of any order via hyperedges of any other order. This approach enables us to probe higher-order structures, define scale invariance at various orders and propose a coarse-graining scheme. We validate our approach on controlled synthetic higher-order systems and then use it to detect the presence of order-specific scale-invariant profiles of real-world complex systems from multiple domains.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">nurisso2025higherorder</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Higher-order Laplacian Renormalization}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nurisso, M. and Morandini, M. and Lucas, M. and Vaccarino, F. and Gili, T. and Petri, G.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Physics}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--8}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41567-025-02784-1}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="santoro2023higher" class="col-sm-8"> <div class="title">Higher-order connectomics of human brain function reveals local topological signatures of task decoding, individual identification, and behavior</div> <div class="author"> A. Santoro, F. Battiston, <em>M. Lucas</em>, G. Petri, and E. Amico </div> <div class="periodical"> <em>Nature Communications</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41467-024-54472-y" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Traditional models of human brain activity often represent it as a network of pairwise interactions between brain regions. Going beyond this limitation, recent approaches have been proposed to infer higher-order interactions from temporal brain signals involving three or more regions. However, to this day it remains unclear whether methods based on inferred higher-order interactions outperform traditional pairwise ones for the analysis of fMRI data. To address this question, we conducted a comprehensive analysis using fMRI time series of 100 unrelated subjects from the Human Connectome Project. We show that higher-order approaches greatly enhance our ability to decode dynamically between various tasks, to improve the individual identification of unimodal and transmodal functional subsystems, and to strengthen significantly the associations between brain activity and behavior. Overall, our approach sheds new light on the higher-order organization of fMRI time series, improving the characterization of dynamic group dependencies in rest and tasks, and revealing a vast space of unexplored structures within human functional brain data, which may remain hidden when using traditional pairwise approaches.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">santoro2023higher</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Higher-order connectomics of human brain function reveals local topological signatures of task decoding, individual identification, and behavior}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Santoro, A. and Battiston, F. and Lucas, M. and Petri, G. and Amico, E.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Communications}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{10244}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{brain}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41467-024-54472-y}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang2024deeper" class="col-sm-8"> <div class="title">Deeper but smaller: Higher-order interactions increase linear stability but shrink basins</div> <div class="author"> Y. Zhang, P. S. Skardal, F. Battiston, G. Petri, and <em>M. Lucas</em> </div> <div class="periodical"> <em>Science Advances</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/sciadv.ado8049" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://github.com/maximelucas/basins_and_triangles" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="abstract hidden"> <p>A key challenge of nonlinear dynamics and network science is to understand how higher-order interactions influence collective dynamics. Although many studies have approached this question through linear stability analysis, less is known about how higher-order interactions shape the global organization of different states. Here, we shed light on this issue by analyzing the rich patterns supported by identical Kuramoto oscillators on hypergraphs. We show that higher-order interactions can have opposite effects on linear stability and basin stability: they stabilize twisted states (including full synchrony) by improving their linear stability, but also make them hard to find by dramatically reducing their basin size. Our results highlight the importance of understanding higher-order interactions from both local and global perspectives.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2024deeper</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Deeper but smaller: Higher-order interactions increase linear stability but shrink basins}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang, Y. and Skardal, P. S. and Battiston, F. and Petri, G. and Lucas, M.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Science Advances}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order_sync}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1126/sciadv.ado8049}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="brondetta2024benefits" class="col-sm-8"> <div class="title">On the Benefits of Heterogeneity in Cognitive Stability and Flexibility for Collaborative Task Switching</div> <div class="author"> A. Brondetta, A. Bizyaeva, <em>M. Lucas</em>, G. Petri, and S. Musslick </div> <div class="periodical"> <em>Proceedings of the Annual Meeting of the Cognitive Science Society</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Environments pose antagonistic demands on individual and collective cognition, such as trading off cognitive stability against cognitive flexibility. Manifestations of this tradeoff have been shown to vary across individuals, leading to differences in individual task switching performance. In this simulation study, we examine how individual differences in cognitive stability and flexibility contribute to collective task switching performance. Specifically, we study whether diversity in cognitive stability and flexibility among members of a group can facilitate collaborative task switching. We test this hypothesis by probing task switching performance of a multi-agent dynamical system, and by varying the heterogeneity of cognitive stability and flexibility among agents. We find that heterogeneous (compared to homogeneous) groups perform better in environments with high switch rates, especially if the most flexible agents receive task switch instructions. We discuss the implications of these findings for normative accounts of cognitive heterogeneity, as well as clinical and educational settings.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">brondetta2024benefits</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{On the Benefits of Heterogeneity in Cognitive Stability and Flexibility for Collaborative Task Switching}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Brondetta, A. and Bizyaeva, A. and Lucas, M. and Petri, G. and Musslick, S.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Proceedings of the Annual Meeting of the Cognitive Science Society}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{46}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://escholarship.org/uc/item/6b47b61g}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="leitao2024evidence" class="col-sm-8"> <div class="title">Evidence of social learning across symbolic cultural barriers in sperm whales</div> <div class="author"> A. Leitão, <em>M. Lucas</em>, S. Poetto, T. A. Hersh, S. Gero, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'D. Gruber, M. Bronstein, G. Petri' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>eLife</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.7554/eLife.96362.1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Social learning is key in the development of both human and non-human animal societies. Here, we provide quantitative evidence that supports the existence of social learning in sperm whales across socio-cultural barriers, based on acoustic data from locations in the Pacific and Atlantic Oceans. Sperm whale populations have traditionally been partitioned into clans based on their vocal repertoire (what they say) of rhythmically patterned clicks (codas), and in particular their identity codas, which serve as symbolic markers for each clan. However, identity codas account for between 35% and 60% of all codas vocalized depending on the different clans. We introduce a computational modeling approach that recovers clan structure and shows new evidence of social learning across clans from the internal temporal structure of non-identity codas, the remaining fraction of codas. The proposed method is based on vocal style, which encodes how sperm whales assemble individual clicks into codas. Specifically, we modeled clicking pattern data using generative models based on variable length Markov chains, producing what we term "subcoda trees". Based on our results, we propose here a new concept of vocal identity, which consists of both vocal repertoire and style. We show that (i) style-delimited clans are similar to repertoire-delimited clans, and that (ii) sympatry increases vocal style similarity between clans for non-identity codas, but has no significant effect on identity codas. This implies that different clans who geographically overlap have similar styles for most codas, which in turn implies social learning across cultural boundaries. More broadly, the proposed method provides a new framework for comparing communication systems of other animal species, with potential implications for our understanding of cultural transmission in animal societies.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">leitao2024evidence</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Evidence of social learning across symbolic cultural barriers in sperm whales}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Leit{\~a}o, A. and Lucas, M. and Poetto, S. and Hersh, T. A. and Gero, S. and Gruber, D. and Bronstein, M. and Petri, G.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{eLife}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{13}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{ceti}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.7554/eLife.96362.1}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="nurisso2024unified" class="col-sm-8"> <div class="title">A unified framework for Simplicial Kuramoto models</div> <div class="author"> M. Nurisso, A. Arnaudon, <em>M. Lucas</em>, R. L. Peach, P. Expert, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'F. Vaccarino, G. Petri' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Chaos</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1063/5.0169388" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Simplicial Kuramoto models have emerged as a diverse and intriguing class of models describing oscillators on simplices rather than nodes. In this paper, we present a unified framework to describe different variants of these models, categorized into three main groups: "simple" models, "Hodge-coupled" models, and "order-coupled" (Dirac) models. Our framework is based on topology, discrete differential geometry as well as gradient flows and frustrations, and permits a systematic analysis of their properties. We establish an equivalence between the simple simplicial Kuramoto model and the standard Kuramoto model on pairwise networks under the condition of manifoldness of the simplicial complex. Then, starting from simple models, we describe the notion of simplicial synchronization and derive bounds on the coupling strength necessary or sufficient for achieving it. For some variants, we generalize these results and provide new ones, such as the controllability of equilibrium solutions. Finally, we explore a potential application in the reconstruction of brain functional connectivity from structural connectomes and find that simple edge-based Kuramoto models perform competitively or even outperform complex extensions of node-based models.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">nurisso2024unified</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A unified framework for Simplicial Kuramoto models}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nurisso, M. and Arnaudon, A. and Lucas, M. and Peach, R. L. and Expert, P. and Vaccarino, F. and Petri, G.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Chaos}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{34}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order_sync}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1063/5.0169388}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2023phasik" class="col-sm-8"> <div class="title">Phasik: a Python package to identify system states in partially temporal networks</div> <div class="author"> <em>M. Lucas</em>, A. Townsend-Teague, M. Neri, S. Poetto, A. Morris, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'B. H. Habermann, L. Tichit' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Journal of Open Source Software</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.21105/joss.05872" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://gitlab.com/habermann_lab/phasik" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="abstract hidden"> <p>Phasik is a Python library for analyzing the temporal structure of temporal and partially temporal networks. Temporal networks are used to model complex systems that consist of entities with time-varying interactions. This library provides methods for building temporal networks (including from data), visualizing them, and analyzing their structure. In particular, Phasik focuses on the identification of temporal phases, that is, periods of time during which the system is in a given state. The library supports partially temporal networks for which information about only a subset of the edges’ temporal evolution is available. Phasik is implemented in pure Python and integrates with the rest of the Python scientific stack.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lucas2023phasik</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.21105/joss.05872}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{The Open Journal}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{91}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{5872}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Townsend-Teague, A. and Neri, M. and Poetto, S. and Morris, A. and Habermann, B. H. and Tichit, L.}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Phasik: a Python package to identify system states in partially temporal networks}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Open Source Software}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{bio}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="landry2023xgi" class="col-sm-8"> <div class="title">XGI: A Python package for higher-order interaction networks</div> <div class="author"> N. W. Landry, <em>M. Lucas</em>, I. Iacopini, G. Petri, A. Schwarze, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'A. Patania, L. Torres' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Journal of Open Source Software</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.21105/joss.05162" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://github.com/ComplexGroupInteractions/xgi" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="abstract hidden"> <p>CompleX Group Interactions (XGI) is a library for analyzing higher-order networks. Such networks are used to model interactions of arbitrary size between entities in a complex system. This library provides methods for building hypergraphs and implicial complexes; algorithms to analyze their structure, visualize them, and simulate dynamical processes on them; and a collection of higher-order datasets. XGI is implemented in pure Python and integrates with the rest of the Python scientific stack. XGI is designed and developed by network scientists with the needs of network scientists in mind.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">landry2023xgi</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Landry, N. W. and Lucas, M. and Iacopini, I. and Petri, G. and Schwarze, A. and Patania, A. and Torres, L.}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.21105/joss.05162}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Open Source Software}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{85}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{5162}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{XGI: A Python package for higher-order interaction networks}}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang2023higher" class="col-sm-8"> <div class="title">Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes</div> <div class="author"> Y. Zhang<sup>*</sup>, <em>M. Lucas<sup>*</sup></em>, and F. Battiston </div> <div class="periodical"> <em>Nature Communications</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41467-023-37190-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a href="http://arxiv.org/abs/2203.03060" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.youtube.com/watch?v=1Hh01jxl1bo" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Video</a> <a href="https://github.com/maximelucas/HOI_shape_sync_differently" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <a href="/assets/pdf/slides/IPAM22.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="abstract hidden"> <p>Higher-order interactions, through which three or more entities interact simultaneously, are important to the faithful modeling of many real-world complex systems. Recent efforts have focused on elucidating the effects of these nonpairwise interactions on the collective behaviors of coupled systems. Interestingly, several examples of higher-order interactions promoting synchronization have been found, raising speculations that this might be a general phenomenon. Here, we demonstrate that even for simple systems such as Kuramoto oscillators, the effects of higher-order interactions are highly nuanced. In particular, we show numerically and analytically that hyperedges typically enhance synchronization in random hypergraphs, but have the opposite effect in simplicial complexes. As an explanation, we identify higher-order degree heterogeneity as the key structural determinant of synchronization stability in systems with a fixed coupling budget. Typical to nonlinear systems, we also capture regimes where pairwise and nonpairwise interactions synergize to optimize synchronization. Our work contributes to a better understanding of dynamical systems with structured higher-order interactions.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zhang2023higher</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zhang*, Y. and Lucas*, M. and Battiston, F.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Communications}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1605}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41467-023-37190-9}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2022simplicially" class="col-sm-8"> <div class="title">Simplicially driven simple contagion</div> <div class="author"> <em>M. Lucas<sup>*</sup></em>, I. Iacopini<sup>*</sup>, T. Robiglio, A. Barrat, and G. Petri </div> <div class="periodical"> <em>Physical Review Research</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1103/PhysRevResearch.5.013201" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a href="http://arxiv.org/abs/2206.07645" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/slides/CSS22_driven.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="abstract hidden"> <p>Single contagion processes are known to display a continuous transition from an epidemic-free phase at low contagion rates to the epidemic state for rates above a critical threshold. This transition can become discontinuous when two simple contagion processes are coupled in a bi-directional symmetric way. However, in many cases, the coupling is not symmetric and the processes can be of a different nature. For example, social behaviors—such as hand-washing or mask-wearing—can affect the spread of a disease, and their adoption dynamics via social reinforcement mechanisms are better described by complex contagion models, rather than by the simple contagion paradigm, which is more appropriate for disease spreading phenomena. Motivated by this example, we consider a simplicial contagion (describing the adoption of a behavior) that uni-directionally drives a simple contagion (describing a disease propagation). We show that, above a critical driving strength, such driven simple contagion can exhibit both discontinuous transitions and bi-stability, which are instead absent in standard simple contagions. We provide a mean-field analytical description of the phase diagram of the system, and complement the results with Markov-chain simulations. Our results provide a novel route for a simple contagion process to display the phenomenology of a higher-order contagion, through a driving mechanism that may be hidden or unobservable in many practical instances.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lucas2022simplicially</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Simplicially driven simple contagion}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas*, M. and Iacopini*, I. and Robiglio, T. and Barrat, A. and Petri, G.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physical Review Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">issue</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{013201}</span><span class="p">,</span>
  <span class="na">numpages</span> <span class="p">=</span> <span class="s">{10}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevResearch.5.013201}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2023inferring" class="col-sm-8"> <div class="title">Inferring cell cycle phases from a partially temporal network of protein interactions</div> <div class="author"> <em>M. Lucas</em>, A. Morris, A. Townsend-Teague, L. Tichit, B. H. Habermann, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'A. Barrat' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em>Cell Reports Methods</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/https://doi.org/10.1016/j.crmeth.2023.100397" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://gitlab.com/habermann_lab/phasik" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <a href="/assets/pdf/slides/netbiomed_2021.pdf" class="btn btn-sm z-depth-0" role="button">Slides</a> </div> <div class="abstract hidden"> <p>The temporal organization of biological systems is key for understanding them, but current methods for identifying this organization are often ad hoc and require prior knowledge. We present Phasik, a method that automatically identifies this multiscale organization by combining time series data (protein or gene expression) and interaction data (protein-protein interaction network). Phasik builds a (partially) temporal network and uses clustering to infer temporal phases. We demonstrate the method’s effectiveness by recovering well-known phases and sub-phases of the cell cycle of budding yeast and phase arrests of mutants. We also show its general applicability using temporal gene expression data from circadian rhythms in wild-type and mutant mouse models. We systematically test Phasik’s robustness and investigate the effect of having only partial temporal information. As time-resolved, multiomics datasets become more common, this method will allow the study of temporal regulation in lesser-known biological contexts, such as development, metabolism, and disease.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lucas2023inferring</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Inferring cell cycle phases from a partially temporal network of protein interactions}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Morris, A. and Townsend-Teague, A. and Tichit, L. and Habermann, B. H. and Barrat, A.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Cell Reports Methods}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{100397}</span><span class="p">,</span>
  <span class="na">biorxiv</span> <span class="p">=</span> <span class="s">{10.1101/2021.03.26.437187v1}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1016/j.crmeth.2023.100397}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{bio}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="newman2021stabilization" class="col-sm-8"> <div class="title">Stabilization of cyclic processes by slowly varying forcing</div> <div class="author"> J. Newman, <em>M. Lucas</em>, and A. Stefanovska </div> <div class="periodical"> <em>Chaos</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1063/5.0066641" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://doi.org/10.17635/lancaster/researchdata/292" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="abstract hidden"> <p>We introduce a new mathematical framework for the qualitative analysis of dynamical stability, designed particularly for finite-time processes subject to slow-timescale external influences. In particular, our approach is to treat finite-time dynamical systems in terms of a slow–fast formalism in which the slow time only exists in a bounded interval, and consider stability in the singular limit. Applying this to one-dimensional phase dynamics, we provide stability definitions somewhat analogous to the classical infinite-time definitions associated with Aleksandr Lyapunov. With this, we mathematically formalize and generalize a phase-stabilization phenomenon previously described in the physics literature for which the classical stability definitions are inapplicable and instead our new framework is required.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">newman2021stabilization</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Stabilization of cyclic processes by slowly varying forcing}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Newman, J. and Lucas, M. and Stefanovska, A.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Chaos}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{31}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{123129}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{AIP Publishing LLC}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1063/5.0066641}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{nonautonomous_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2020multi" class="col-sm-8"> <div class="title">Multiorder Laplacian for synchronization in higher-order networks</div> <div class="author"> <em>M. Lucas</em>, G. Cencetti, and F. Battiston </div> <div class="periodical"> <em>Physical Review Research</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1103/PhysRevResearch.2.033410" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a href="http://arxiv.org/abs/2003.09734" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://github.com/maximelucas/multiorder_Laplacian" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="abstract hidden"> <p>The emergence of synchronization in systems of coupled agents is a pivotal phenomenon in physics, biology, computer science, and neuroscience. Traditionally, interaction systems have been described as networks, where links encode information only on the pairwise influences among the nodes. Yet, in many systems, interactions among the units take place in larger groups. Recent work has shown that the presence of higher-order interactions between oscillators can significantly affect the emerging dynamics. However, these early studies have mostly considered interactions up to four oscillators at time, and analytical treatments are limited to the all-to-all setting. Here, we propose a general framework that allows us to effectively study populations of oscillators where higher-order interactions of all possible orders are considered, for any complex topology described by arbitrary hypergraphs, and for general coupling functions. To this end, we introduce a multiorder Laplacian whose spectrum determines the stability of the synchronized solution. Our framework is validated on three structures of interactions of increasing complexity. First, we study a population with all-to-all interactions at all orders, for which we can derive in a full analytical manner the Lyapunov exponents of the system, and for which we investigate the effect of including attractive and repulsive interactions. Second, we apply the multiorder Laplacian framework to synchronization on a synthetic model with heterogeneous higher-order interactions. Finally, we compare the dynamics of coupled oscillators with higher-order and pairwise couplings only, for a real dataset describing the macaque brain connectome, highlighting the importance of faithfully representing the complexity of interactions in real-world systems. Taken together, our multiorder Laplacian allows us to obtain a complete analytical characterization of the stability of synchrony in arbitrary higher-order networks, paving the way toward a general treatment of dynamical processes beyond pairwise interactions.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lucas2020multi</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Multiorder Laplacian for synchronization in higher-order networks}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Cencetti, G. and Battiston, F.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physical Review Research}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{033410}</span><span class="p">,</span>
  <span class="na">numpages</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevResearch.2.033410}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="battiston2020networks" class="col-sm-8"> <div class="title">Networks beyond pairwise interactions: structure and dynamics</div> <div class="author"> F. Battiston, G. Cencetti, I. Iacopini, V. Latora, <em>M. Lucas</em>, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'A. Patania, J.-G. Young, G. Petri' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Physics Reports</em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.physrep.2020.05.004" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a href="http://arxiv.org/abs/2006.01764" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>The complexity of many biological, social and technological systems stems from the richness of the interactions among their units. Over the past decades, a variety of complex systems has been successfully described as networks whose interacting pairs of nodes are connected by links. Yet, from human communications to chemical reactions and ecological systems, interactions can often occur in groups of three or more nodes and cannot be described simply in terms of dyads. Until recently little attention has been devoted to the higher-order architecture of real complex systems. However, a mounting body of evidence is showing that taking the higher-order structure of these systems into account can enhance our modeling capacities and help us understand and predict their dynamical behavior. Here we present a complete overview of the emerging field of networks beyond pairwise interactions. We discuss how to represent higher-order interactions and introduce the different frameworks used to describe higher-order systems, highlighting the links between the existing concepts and representations. We review the measures designed to characterize the structure of these systems and the models proposed to generate synthetic structures, such as random and growing bipartite graphs, hypergraphs and simplicial complexes. We introduce the rapidly growing research on higher-order dynamical systems and dynamical topology, discussing the relations between higher-order interactions and collective behavior. We focus in particular on new emergent phenomena characterizing dynamical processes, such as diffusion, synchronization, spreading, social dynamics and games, when extended beyond pairwise interactions. We conclude with a summary of empirical applications, and an outlook on current modeling and conceptual frontiers.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">battiston2020networks</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Networks beyond pairwise interactions: structure and dynamics}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Battiston, F. and Cencetti, G. and Iacopini, I. and Latora, V. and Lucas, M. and Patania, A. and Young, J.-G. and Petri, G.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physics Reports}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{874}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--92}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.physrep.2020.05.004}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2019nonautonomous" class="col-sm-8"> <div class="title">Nonautonomous driving induces stability in network of identical oscillators</div> <div class="author"> <em>M. Lucas</em>, D. Fanelli, and A. Stefanovska </div> <div class="periodical"> <em>Physical Review E</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1103/PhysRevE.99.012309" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a href="http://arxiv.org/abs/1809.04833" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Nonautonomous driving of an oscillator has been shown to enlarge the Arnold tongue in parameter space, but little is known about the analogous effect for a network of oscillators. To test the hypothesis that deterministic nonautonomous perturbation is a good candidate for stabilizing complex dynamics, we consider a network of identical phase oscillators driven by an oscillator with a slowly time-varying frequency. We investigate both the short- and long-term stability of the synchronous solutions of this nonautonomous system. For attractive couplings we show that the region of stability grows as the amplitude of the frequency modulation is increased, through the birth of an intermittent synchronization regime. For repulsive couplings, we propose a control strategy to stabilize the dynamics by altering very slightly the network topology. We also show how, without changing the topology, time-variability in the driving frequency can itself stabilize the dynamics. As a byproduct of the analysis, we observe chimeralike states. We conclude that time-variability-induced stability phenomena are also present in networks, reinforcing the idea that this is a quite realistic scenario for living systems to use in maintaining their functioning in the face of ongoing perturbations.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lucas2019nonautonomous</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Nonautonomous driving induces stability in network of identical oscillators}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Fanelli, D. and Stefanovska, A.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physical Review E}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{99}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{012309}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.99.012309}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{nonautonomous_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2018desynchronization" class="col-sm-8"> <div class="title">Desynchronization induced by time-varying network</div> <div class="author"> <em>M. Lucas</em>, D. Fanelli, T. Carletti, and J. Petit </div> <div class="periodical"> <em>Europhys. Lett.</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1209/0295-5075/121/50008" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a href="http://arxiv.org/abs/1802.06580" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>The synchronous dynamics of an array of excitable oscillators, coupled via a generic graph, is studied. Non-homogeneous perturbations can grow and destroy synchrony, via a self-consistent instability which is solely instigated by the intrinsic network dynamics. By acting on the characteristic time-scale of the network modulation, one can make the examined system to behave as its (partially) averaged analogue. This result is formally obtained by proving an extended version of the averaging theorem, which allows for partial averages to be carried out. As a byproduct of the analysis, oscillation death is reported to follow the onset of the network-driven instability.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lucas2018desynchronization</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Desynchronization induced by time-varying network}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Fanelli, D. and Carletti, T. and Petit, J.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Europhys. Lett.}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{121}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{50008}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{IOP Publishing}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1209/0295-5075/121/50008}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{nonautonomous_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2018stabilization" class="col-sm-8"> <div class="title">Stabilization of dynamics of oscillatory systems by nonautonomous perturbation</div> <div class="author"> <em>M. Lucas</em>, J. Newman, and A. Stefanovska </div> <div class="periodical"> <em>Physical Review E</em>, 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1103/PhysRevE.97.042209" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a href="http://arxiv.org/abs/1803.11392" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Synchronization and stability under periodic oscillatory driving are well understood, but little is known about the effects of aperiodic driving, despite its abundance in nature. Here, we consider oscillators subject to driving with slowly varying frequency, and investigate both short-term and long-term stability properties. For a phase oscillator, we find that, counterintuitively, such variation is guaranteed to enlarge the Arnold tongue in parameter space. Using analytical and numerical methods that provide information on time-variable dynamical properties, we find that the growth of the Arnold tongue is specifically due to the growth of a region of intermittent synchronization where trajectories alternate between short-term stability and short-term neutral stability, giving rise to stability on average. We also present examples of higher-dimensional nonlinear oscillators where a similar stabilization phenomenon is numerically observed. Our findings help support the case that in general, deterministic nonautonomous perturbation is a very good candidate for stabilizing complex dynamics.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lucas2018stabilization</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Stabilization of dynamics of oscillatory systems by nonautonomous perturbation}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Newman, J. and Stefanovska, A.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Physical Review E}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{97}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{042209}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2018}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1103/PhysRevE.97.042209}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{nonautonomous_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="duncan2014verifying" class="col-sm-8"> <div class="title">Verifying the Steane code with Quantomatic</div> <div class="author"> R. Duncan and <em>M. Lucas</em> </div> <div class="periodical"> <em>Electronic Proceedings in Theoretical Computer Science</em>, 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.4204/EPTCS.171.4" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>In this paper we give a partially mechanized proof of the correctness of Steane’s 7-qubit error correcting code, using the tool Quantomatic. To the best of our knowledge, this represents the largest and most complicated verification task yet carried out using Quantomatic.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">duncan2014verifying</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Duncan, R. and Lucas, M.}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Electronic Proceedings in Theoretical Computer Science}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{171}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{33--49}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2014}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.4204/EPTCS.171.4}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <h2 id="book-chapters">book chapters</h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2022multiorder" class="col-sm-8"> <div class="title">Multiorder Laplacian for Kuramoto Dynamics with Higher-Order Interactions</div> <div class="author"> <em>M. Lucas</em>, G. Cencetti, and F. Battiston </div> <div class="periodical"> <em>In Higher-Order Systems</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/978-3-030-91374-8_9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Many real-world systems are characterised by higher-order interactions, where influences among units involve more than two nodes at a time, and which can significantly affect the emergence of collective behaviors. A paradigmatic case is that of synchronization, occuring when oscillators reach coherent dynamics through their mutual couplings, and which is known to display richer collective phenomena when connections are not limited to simple dyads. Here, we consider an extension of the Kuramoto model with higher-order interactions, where oscillators can interact in groups of any size, arranged in any arbitrary complex topology. We present a new operator, the multiorder Laplacian, which allows us to treat the system analytically and that can be used to assess the stability of synchronization in general higher-order networks. Our spectral approach, originally devised for Kuramoto dynamics, can be extended to a wider class of dynamical processes beyond pairwise interactions, advancing our quantitative understanding of how higher-order interactions impact network dynamics.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@incollection</span><span class="p">{</span><span class="nl">lucas2022multiorder</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Multiorder Laplacian for Kuramoto Dynamics with Higher-Order Interactions}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Cencetti, G. and Battiston, F.}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{233}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Higher-Order Systems}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{9783030913748}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer Nature}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-3-030-91374-8_9}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{higher-order_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2021synchronisation" class="col-sm-8"> <div class="title">Synchronisation and Non-autonomicity</div> <div class="author"> <em>M. Lucas</em>, J. Newman, and A. Stefanovska </div> <div class="periodical"> <em>In Physics of Biological Oscillators</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/978-3-030-59805-1_6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Thermodynamic openness is key to the long-term stability of living systems and can yield rich dynamical behaviours. Here, we model openness in coupled oscillator systems by introducing an external driving with time-varying parameters. Five systems of increasing complexity are considered: three cases of single driven oscillators followed by two cases of driven networks. We show how the time-varying parameters can enlarge the range of other parameters for which synchronous behaviour is stable. In addition, it can yield additional behaviours such as intermittent synchronisation. The stability of these systems is analysed via short- and long-time Lyapunov exponents, both analytically and numerically. The different dynamical regimes are also described via time-frequency representation. Finally, we compare the stabilising effect of deterministic non-autonomous driving to that of bounded noise. All in all, we give an overview of some effects time-varying parameters can have on synchronisation. These effects could be a key to understand how living systems maintain stability in the face of their ever-changing environment.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@incollection</span><span class="p">{</span><span class="nl">lucas2021synchronisation</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Synchronisation and Non-autonomicity}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M. and Newman, J. and Stefanovska, A.}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Physics of Biological Oscillators}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{85--110}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-3-030-59805-1_6}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{nonautonomous_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="newman2021non" class="col-sm-8"> <div class="title">Non-asymptotic-time Dynamics</div> <div class="author"> J. Newman, <em>M. Lucas</em>, and A. Stefanovska </div> <div class="periodical"> <em>In Physics of Biological Oscillators</em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1007/978-3-030-59805-1_7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="abstract hidden"> <p>Traditional analysis of dynamics concerns coordinate-invariant features of the long-time-asymptotic behaviour of a system. Using the non-autonomous Adler equation with slowly varying forcing, we illustrate three of the limitations of this traditional approach. We discuss an alternative, “slow-fast finite-time dynamical systems” approach, that is more suitable for slowly time-dependent one-dimensional phase dynamics, and is likely to be suitable for more general dynamics of open systems involving two or more timescales.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@incollection</span><span class="p">{</span><span class="nl">newman2021non</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Non-asymptotic-time Dynamics}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Newman, J. and Lucas, M. and Stefanovska, A.}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Physics of Biological Oscillators}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{111--129}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-3-030-59805-1_7}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{nonautonomous_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <h2 id="theses">theses</h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2019synchronisation" class="col-sm-8"> <div class="title">Synchronisation and stability in nonautonomous oscillatory systems</div> <div class="author"> <em>M. Lucas</em> </div> <div class="periodical"> <em>Lancaster University and University of Florence</em>, 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.17635/lancaster/thesis/502" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/2019phdlucas.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Many natural and artificial systems can be modelled by ensembles of coupled oscillators. These types of systems can exhibit various synchronisation phenomena, where the interaction between the oscillators leads them to some kind of coherent behaviour, despite heterogeneities in the system. Moreover, many such systems are subject to a timevariable environment which effectively drives them. Many examples can be found in living systems, e.g., the dynamics of a cell is strongly dependent on the ever-changing intra- and extra-cellular ionic concentrations.Motivated by these considerations, this thesis investigates the effect of time-varying parameters on synchronisation and stability in ensembles of coupled oscillators. Timevariability is a crucial ingredient of the dynamics of many real-life systems, and interest in it is only recently starting to grow. Such systems are in general described by nonautonomous equations, which are hard to treat in general. This present work aims at answering questions such as: Can time-variability be detrimental/beneficial to synchronisation? If so, under which conditions? Can time-variability seed new dynamical phenomena? How can one best treat nonautonomous systems?The systems studied can be divided into two categories. First, the effect of a driving oscillator with a time-varying frequency is investigated. It is shown that increasing the amplitude of the frequency modulation can increase the size of the stability region in parameter space, under general assumptions. Short-term dynamics and stability properties are also investigated, and their dynamics is shown to be of importance. Second, the effect of time-varying couplings between the oscillators is considered. This is shown to be able to make the synchronous state unstable and yield oscillation death.Overall, the thesis illustrates that time-variability can be either beneficial or detrimental to synchronous dynamics, and investigates in detail and gives insight about cases of both. It argues towards the general fact that short-term dynamics is often crucial to a physically relevant understanding of nonautonomous systems.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@phdthesis</span><span class="p">{</span><span class="nl">lucas2019synchronisation</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Synchronisation and stability in nonautonomous oscillatory systems}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lucas, M.}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2019}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.17635/lancaster/thesis/502}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Lancaster University}</span><span class="p">,</span>
  <span class="na">school</span> <span class="p">=</span> <span class="s">{Lancaster University and University of Florence}</span><span class="p">,</span>
  <span class="na">project</span> <span class="p">=</span> <span class="s">{nonautonomous_sync}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2015instabilities" class="col-sm-8"> <div class="title">Instabilities in cortical networks with embedded synfire chains</div> <div class="author"> <em>M. Lucas</em> </div> <div class="periodical"> <em>Katholieke Universiteit Leuven</em>, 2015 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="lucas2014instabilites" class="col-sm-8"> <div class="title">Instabilités dynamiques de systèmes de billards avec interactions rares</div> <div class="author"> <em>M. Lucas</em> </div> <div class="periodical"> <em>Université Libre de Bruxelles</em>, 2014 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 Maxime Lucas. Based on the <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Last updated: October 01, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> </body> </html>